VideoWeb/VideoWeb.UnitTests/Mappings/ParticipantForUserResponseMapperTests.cs

```diff
@@ -95,7 +102,7 @@ public void Should_map_all_participants()
 
                 if (position[0].StartsWith("CIVILIAN") || position[0].StartsWith("WITNESS"))
                 {
-                    tiledNames.Count(x => x.StartsWith(position[0])).Should().Be(5);
+                    tiledNames.Count(x => x.StartsWith(position[0])).Should().Be(6);
```

Comment:
Can this be figured out instead of needing to be changed in the test every time we update?